src/workspace.coffee

```diff
@@ -47,6 +47,8 @@ class Workspace extends Model
       left: new PanelContainer({location: 'left'})
       right: new PanelContainer({location: 'right'})
       bottom: new PanelContainer({location: 'bottom'})
+      topBar: new PanelContainer({location: 'top-bar'})
+      bottomBar: new PanelContainer({location: 'bottom-bar'})
```

Comment:
`header` and `footer` are my favorites, especially since the terms are extremely common in the web world and we're using web tech in atom :)